ppoprf/examples/server.rs

```diff
@@ -141,20 +141,28 @@ async fn main() -> std::io::Result<()> {
         prf_server: ppoprf::Server::new(&mds),
         md_idx: 0
     }));
+    info!("PPOPRF initialized with epoch metadata tags {:?}", &mds);
 
     // Spawn a background task.
     let background_state = state.clone();
     actix_web::rt::spawn(async move {
-        log::info!("Spawned background epoch rotation task");
+        info!("Background task will rotate epoch every {} seconds", epoch.as_secs());
         // Wait for the end of an epoch.
         for md in &mds {
             actix_web::rt::time::delay_for(epoch).await;
             if let Ok(mut state) = background_state.write() {
-                log::info!("Epoch rotation: puncturing '{:?}'", md);
+                info!("Epoch rotation: puncturing '{:?}'", md);
                 state.prf_server.puncture(md);
                 state.md_idx += 1;
             }
-            log::info!("Epoch rotation complete");
+            if let Ok(state) = background_state.read() {
+                if state.md_idx >= mds.len() {
+                    warn!("All epoch tags punctured! No further evaluations possible.");
```

Comment:
what if we placed this warning log right after the loop...

Reply:
That's a little cleaner, thanks.